docs-src/guidelines.md

```diff
@@ -83,3 +83,12 @@ We expect there will be varying levels of conformance among the
 different providers in the early days of this API. Users can use the
 results of the conformance tests to understand areas where there may
 be differences in behavior from the spec.
+
+## API Conventions
+
+Service APIs follow Kubernetes API [conventions][1]. These conventions
+are intended to ease client development and ensure that configuration
+mechanisms can consistently be implemented across a diverse set of use
+cases.
+
+[1]: https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md
```

Comment:
I think the way we're treating fields with default values may be slightly unique here and worth a callout. If I'm understanding the approach correctly, fields with default values are considered optional but will not be pointers because they will never truly be empty?

Reply:
@robscott I've updated, PTAL.